commands/permissions.go

```diff
@@ -109,26 +173,147 @@ func showRoleCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		return response.Error
 	}
 
-	tpl := `Name: {{.Name}}
-Display Name: {{.DisplayName}}
-Description: {{.Description}}
-Permissions: {{.Permissions}}
-{{range .Permissions}}
-  - {{.}}
-{{end}}
-{{if .BuiltIn}}
-Built in: yes
-{{else}}
-Built in: no
-{{end}}
-{{if .SchemeManaged}}
-Scheme Managed: yes
-{{else}}
-Scheme Managed: no
-{{end}}
-`
-
-	printer.PrintT(tpl, role)
+	sort.Strings(role.Permissions)
+
+	consolePermissionMap := map[string]bool{}
+	for _, perm := range role.Permissions {
+		if strings.HasPrefix(perm, "sysconsole_") {
+			consolePermissionMap[perm] = true
+		}
+	}
+
+	getUsedBy := func(permissionID string) []string {
+		var usedByIDs []string
+		if !strings.HasPrefix(permissionID, "sysconsole_") {
+			usedBy := map[string]bool{} // map to make a unique set
+			for key, vals := range model.SysconsoleAncillaryPermissions {
+				for _, val := range vals {
+					if val.Id == permissionID {
+						if _, ok := consolePermissionMap[key]; ok {
+							usedBy[key] = true
+						}
+					}
+				}
+			}
+			for key := range usedBy {
+				usedByIDs = append(usedByIDs, key)
+			}
+		}
+		return usedByIDs
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
+
+	// Only show the 3-column view if the role has sysconsole permissions
+	// sysadmin has every permission, so no point in showing the "Used by"
+	// column.
+	if len(consolePermissionMap) > 0 && role.Name != "system_admin" {
+		fmt.Fprintf(w, "Property\tValue\tUsed by\n")
```

Comment:
These prints should be moved to a template, so they can be printed with a `printer.PrintT(tpl, role)` for the `--format json` flag to work properly

nit: I'd suggest to print a `\n` character just before the headings, to leave some space between the command and the plain output:
```suggestion
		fmt.Fprintf(w, "\nProperty\tValue\tUsed by\n")
```